compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -7606,6 +7626,21 @@ private STNode parseAtToken() {
         }
     }
 
+    /**
+     * Parse '*' token.
+     *
+     * @return Parsed node
+     */
+    private STNode parseAsteriskToken() {
+        STToken nextToken = peek();
+        if (nextToken.kind == SyntaxKind.ASTERISK_TOKEN) {
+            return consume();
+        } else {
+            recover(nextToken, ParserRuleContext.ASTERISK);
+            return parseAsteriskToken();
+        }
+    }
```

Comment:
Related to above comment.
```suggestion
    private STNode parseInclusionSymbol() {  
        STToken nextToken = peek();
        if (nextToken.kind == SyntaxKind.ASTERISK_TOKEN) {
            return consume();
        } else {
            return STNodeFactory.createEmptyNode();
        }
    }
```

Reply:
Fixed.